src/network/bridge.rs

```diff
@@ -95,6 +97,13 @@ impl driver::NetworkDriver for Bridge<'_> {
         self.data = Some(InternalData {
             bridge_interface_name: bridge_name,
             container_interface_name: self.info.per_network_opts.interface_name.clone(),
+            host_interface_name: self
+                .info
+                .per_network_opts
+                .options
+                .get("host_interface_name")
+                .cloned()
+                .unwrap_or_else(|| "".to_string()),
```

Comment:
please use parse_option() as split out host_interface_name as constant like the other options

I know parsing a String cannot fail so the error handling is not needed technically but I think it better to use it consistently.

Reply:
Fixed. Do we need to make clear which options come from the network and which from the per-network-config? With the current code, the messages and variables look the same for both kinds of options.